### PR TITLE
Feature/three explainers reformat

### PR DESCRIPTION
**Pull request by frapperino**

### CODE REVIEW

File: src/sections/landing/three-explainers.tsx [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+    marginRight: '30px',
+  },
+  '@media (min-width: 640px)': {
+    fontSize: '18px',
+    lineHeight: '30px',
+  },
+});
+
+class ThreeExplainers extends React.Component<Props> {
+  insuranceInMinutesAnim: any | null = null;
```
Comment (palmfjord) [inline src/sections/landing/three-explainers.tsx]: `any` :/ Maybe add a `TODO` comment about that we should change this type when we get `react-lottie` types into definitely typed

File: src/sections/landing/three-explainers.tsx [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+      <BackgroundWrapper>
+        <Container className="Container">
+          <Header>{this.props.heading}</Header>
+          <VisibilitySensor
+            partialVisibility
+            onChange={this.threeExplainersVisbilityChanged}
+          >
+            <Row>
+              <Col>
+                {/* style={{width: this.getWidth()}} - Does not make any difference */}
```
Comment (palmfjord) [inline src/sections/landing/three-explainers.tsx]: commented out code :/

File: src/sections/landing/three-explainers.tsx [hunk 3]
```diff
@@ -58,6 +58,7 @@ const Row = styled('div')({
 const Col = styled('div')({
   '@media (min-width: 960px)': {
     flexBasis: 'auto',
+    width: 'calc(100% * 1 / 3)',
```
Comment (palmfjord) [inline src/sections/landing/three-explainers.tsx]: wouldn't `100% / 3` be sufficient?

**Comment by hedvigoscar** [comment]
@frapperino Lets get this merged!

**Pull request closed without merging**